docs/DeveloperGuide.md

```diff
@@ -287,7 +287,6 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | Seasoned User  | see which of my friends I am closest to and which of them I have not visited in a while | reflect more about how I spend my time with my friend and maybe catch up with ones I have not met in awhile | 
 | `* *`    | Seasoned User  | chain commands                                                                          | reduce the number of commands I need to type thereby increasing efficiency                                  |
 | `*`      | User           | tag friends under different categories                                                  | better classify and organise my friend circle                                                               | 
-| `*`      | New User       | know which features of the application I am not using as often                          | fully utilise the application to its maximum capability                                                     | 
```

Comment:
Uhh, you can undo your change here cause Dione updated all the user stories already! (PR alr merged)